malicious/pypi/equests/MAL-0000-checkmarx-b07d61adac5cc418.json

```diff
@@ -0,0 +1,70 @@
+{
+  "modified": "2022-05-31T20:12:58Z",
+  "published": "2022-05-31T00:00:00Z",
+  "schema_version": "1.5.0",
+  "id": "",
+  "summary": "Malicious code in equests (PyPi)",
+  "details": "Malicious packages typosquatting the popular requests package. payload execute a cryptomining malware",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "PyPi",
```

Comment:
Can these be changed to "PyPI" as per the OSV standard.

I'll add some validation on these soon.

Reply:
I edited the branch to avoid the round-trip.